nixos/modules/services/misc/overseerr.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.overseerr;
+in
+{
+  meta.maintainers = [ ];
+
+  options.services.overseerr = {
```

Comment:
I'm a bit confused about how this knows how to connect to a plex server. I see there's some documentation about configuration, but it's not entirely clear to me whether it's a configuration file, envvars or some kind of interactive panel with state in a file.

If there's a configuration file available, I think it would be essential to have a `settings` option. And if it's saving state from a web panel to files, I'm a little bit unsure about both setting the working directory into the package, and bindmounting `var/lib/overseerr` into the package.

Have you dogfooded this module?